### PR TITLE
Update to 1.1.0

### PR DESCRIPTION
**Pull request by Jako**
- Agenda plugin events to create/remove Commerce products during Agenda event creation

### CODE REVIEW

File: _build/build.transport.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ function getSnippetContent($filename = '') {
     /* define version */
     define('PKG_NAME', 'Commerce_Cursus');
     define('PKG_NAMESPACE', 'commerce_cursus');
-    define('PKG_VERSION', '0.1.0');
+    define('PKG_VERSION', '1.1.1');
```
Comment (Mark-H) [inline _build/build.transport.php:26]: In general, please don't bump versions in PRs. We may release it as a different number or end up bundle changes together.

File: core/components/commerce_cursus/model/commerce_cursus/commerce_cursus.class.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * @package commerce_cursus
+ * @subpackage classfile
+ */
+
+require_once dirname(__DIR__, 2) . '/vendor/autoload.php';
+
+/**
+ * Class Commerce_Cursus
+ */
+class Commerce_Cursus extends \modmore\Commerce_Cursus\Commerce_Cursus
```
Comment (Mark-H) [inline core/components/commerce_cursus/model/commerce_cursus/commerce_cursus.class.php:16]: What's the purpose of this new empty file that just defines an alias? There's nothing else in `/model/` so I'm confused why we'd need this.

Comment (Jako) [reply core/components/commerce_cursus/model/commerce_cursus/commerce_cursus.class.php:16]: The class can be loaded in MODX2 with getService but maybe it is not needed.

Comment (Mark-H) [reply core/components/commerce_cursus/model/commerce_cursus/commerce_cursus.class.php:16]: Removed in https://github.com/modmore/Commerce_Cursus/commit/81bdcd604f1f6ed6944853b4c24ea1ea6b77488b

**Comment by Jako** [comment]
It seems that loading the service class is needed in the plugin code:

https://github.com/modmore/Commerce_Cursus/blob/main/core/components/commerce_cursus/elements/plugins/commerce_cursus.plugin.php#L16

Maybe this can be solved different, but the service class is missing there

```
[2024-05-01 12:52:22] (ERROR @ /…/core/xpdo/xpdo.class.php : 644) Could not load class: Commerce_Cursus from commerce_cursus.
[2024-05-01 12:52:22] (ERROR @ /…/core/xpdo/xpdo.class.php : 1248) Problem getting service commerce_cursus, instance of class Commerce_Cursus, from path /…/core/components/commerce_cursus/model/commerce_cursus/
```

**Comment by Mark-H** [comment]
It's also supposedly being loaded in the [plugin code](https://github.com/modmore/Commerce_Cursus/blob/main/core/components/commerce_cursus/src/Plugins/Plugin.php#L37) but I don't see any uses of it.

**Comment by Jako** [comment]
Then the service class initialisation has to be removed from the plugin code. Otherwise the plugin code is never executed.

**Comment by Jako** [comment]
Are there any configuration options that can be retrieved from a commerce_cursus service class? Otherwise both initialisations can be removed. In my Ferienprogramm code I don't use them. And other examples don't exist yet.

**Comment by Mark-H** [comment]
I've never used a service class in a Commerce extension, this is code you added. I've only left most of it in because I wasn't sure if there may've been uses I overlooked. 

The [module class](https://github.com/modmore/Commerce_Cursus/blob/main/core/components/commerce_cursus/src/Module.php) is were configuration often comes from.

**Comment by Jako** [comment]
Hmm, the plugin code is running independent from Commerce. So the commerce_cursus classes have to be autoloaded somewhere. I have done this in commerce_cursus.class.php and in the plugin code the service class was initialised.

I think commerce_cursus.class.php has to be restored or the autoloading has to happen in the plugin code. What do you prefer?

**Comment by Jako** [comment]
I have prepared a PR #3 for this.

**Pull request merged**